src/java.base/share/classes/java/util/spi/AbstractResourceBundleProvider.java

```diff
@@ -78,11 +75,11 @@
  *         return null;
  *     }
  * }}</pre></blockquote>
- *
+ * <p>
  * Refer to {@link ResourceBundleProvider} for details.
  *
- * @see <a href="../ResourceBundle.html#resource-bundle-modules">
- *      Resource Bundles and Named Modules</a>
+ * @see <a href="ResourceBundle##resource-bundle-modules">
+ * Resource Bundles and Named Modules</a>
```

Comment:
I think HTML tag is not needed:
```suggestion
 * @see ResourceBundle##resource-bundle-modules
 * Resource Bundles and Named Modules
```